sphinx/docserver.py

```diff
@@ -1,89 +1,84 @@
-import asyncio
-import os
-import sys
+# -----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
```

Comment:
It would be great to have a short method docstring to explain what this script does, and especially why this docserver is required/helpful when displaying the docs.

Reply:
done